src/shaders/MaterialX/mx_globals.h

```diff
@@ -0,0 +1,4 @@
+//defines whether or no the v texture coordinate will be flipped 
+#define FLIP_V 1
```

Comment:
I'm not sure this can or should be handled in the shaders.
There's a ShadingSystem-wide option that you can use called "flip_t" that will do this more efficiently and correctly inside the texture lookups themselves. A studio who prefers that t=0 is the visual bottom of its textures should turn on this option globally.